backtrace-library/src/main/java/backtraceio/library/common/RequestHelper.java

```diff
@@ -0,0 +1,53 @@
+package backtraceio.library.common;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import backtraceio.library.logger.BacktraceLogger;
+
+public class RequestHelper {
+    private static final transient String LOG_TAG = RequestHelper.class.getSimpleName();
+
+    private static final String CRLF = "\r\n";
```

Comment:
Use existing constant in java libs?

Reply:
it's a small nit - I see java has `System.lineSeparator()`

Reply:
System.lineSeparator will return `\n` in Android (Unix), but we need `\r\n` for HTTP request. We use this same style in @BartoszLitwiniuk code:
https://github.com/backtrace-labs/backtrace-android/blob/dcf0eb97675d693c5aff0f733d407197e9dad168/backtrace-library/src/main/java/backtraceio/library/common/MultiFormRequestHelper.java#L18

Reply:
Correct me if I'm wrong but in Android SDK there is not const/enum for `\r\n`, btw Konrad remember that not in all cases Android == Java ;)

Reply:
There is no existing constant for this, I will mark this resolved @vlussenburg @konraddysput